.github/workflows/ci.yaml

```diff
@@ -131,3 +132,96 @@ jobs:
         with:
           name: kind-logs-${{ matrix.ip-family }}-${{ matrix.bgp-type }}-${{ matrix.deployment}}
           path: /tmp/kind_logs/
+
+  e2e-use-operator:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: metallb
+    env:
+      metallb_operator_image: "metallb-operator:ci"
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout Metal LB Operator
+        uses: actions/checkout@v2
+        with:
+          repository: metallb/metallb-operator
+          path: metallboperator
+          ref: a0270bf3264f8e58c20db0b49881e013e0aa8741
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.16.3'
+
+      - name: Verify modules
+        run: |
+          cd ${GITHUB_WORKSPACE}/metallboperator
+          go mod verify
+
+      - name: Verify format
+        run: |
+          cd ${GITHUB_WORKSPACE}/metallboperator
+          make fmt
+          git diff --exit-code
+
+      - name: Verify manifests
+        run: |
+          cd ${GITHUB_WORKSPACE}/metallboperator
+          make manifests
+          git diff --exit-code
+
+      - name: Checkout MetalLB
+        uses: actions/checkout@v2
+        with:
+          path: metallb
+          fetch-depth: 0
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install python3-pip arping ndisc6
+          sudo pip3 install invoke semver pyyaml
```

Comment:
this can be done using with 
`pip3 install --user -r ./dev-env/requirements.txt` the in case we add more dependencies.

Reply:
done